packages/blade/src/components/Radio/RadioIcon/Fade.native.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React from 'react';
+import type { EasingFn } from 'react-native-reanimated';
```

Comment:
```suggestion
import type { EasingFunctionFactory } from 'react-native-reanimated';
```

Reply:
Can't do that. Keyframe() takes EasingFn only.